src/main/java/com/epam/flightscontroltool/domain/airplane/entity/Company.java

```diff
@@ -0,0 +1,25 @@
+package com.epam.flightscontroltool.domain.airplane.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+public class Company {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String name;
+
+    @OneToMany(mappedBy = "company", fetch = FetchType.LAZY, cascade = CascadeType.ALL)
```

Comment:
Well, it's questionable. On the one hand we can assume that an airplane can belong to one company today (e.g. Turkish Airlines) but tomorrow it can be a different company (e.g. Belavia) especially if we are talking about plane leasing. In this case it might be better to keep company (like 'перевозчик') as a field of Flight entity. On the other hand, such changes won't happen every single day, that's why I can leave this connection here (in case it's necessary to change the company to which the airplane belongs we can use 'update' method in AirplaneEndpoint)